packages/shared/src/components/cards/Card.tsx

```diff
@@ -56,19 +56,19 @@ export const CardLink = classed('a', clickableCardClasses);
 export const Card = classed(
   'article',
   styles.card,
-  'snap-start relative max-h-cardLarge h-full flex flex-col p-2 rounded-16 bg-background-subtle border border-border-subtlest-tertiary hover:border-border-subtlest-secondary shadow-2',
```

Comment:
Ignore the header*

Before:
![Uploading Screenshot 2024-09-06 at 7.46.38 PM.png…]()


After:
![Screenshot 2024-09-06 at 7 46 45 PM](https://github.com/user-attachments/assets/1ccb4399-af07-4906-b943-c42291fe40ab)
0QXQJNB/p1725538654411249

Figma:

![Screenshot 2024-09-06 at 8 13 13 PM](https://github.com/user-attachments/assets/a546fe32-b8ee-495a-8826-a7284ed0df57)

Thankfully Tomer mentioned that the shadow should not actually be there generally.
https://dailydotdev.slack.com/archives/C01L